src/main/resources/application.yml

```diff
@@ -73,7 +73,16 @@ workspace:
     uri: ${env.db.host}/${env.db.ws.name}
     username: ${env.db.ws.user}
 
-terra.common:
-  tracing:
-    stackdriverExportEnabled: ${env.tracing.exportEnabled}
-    samplingRate: ${env.tracing.samplingRate}
+terra:
+  common:
+    tracing:
+        stackdriverExportEnabled: ${env.tracing.exportEnabled}
+        samplingRate: ${env.tracing.samplingRate}
+    stairway:
+      cluster-name-suffix: workspace-stairway
+      force-clean-start: false
+      max-parallel-flights: 50
+      migrate-upgrade: false
```

Comment:
migrate-upgrade should be true in non-testing env. force-clean-start should be `${env.db.init}`
See line 59 and 60 above, and you should remove those two lines because they are not used anymore. 
Also those configs should be deleted as well:
https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/src/main/java/bio/terra/workspace/app/configuration/external/StairwayDatabaseConfiguration.java#L13-L42
Those two are replaced by your new code: https://github.com/DataBiosphere/terra-common-lib/blob/develop/src/main/java/bio/terra/common/stairway/StairwayProperties.java#L19-L24